plan9/pdcplan9.c

```diff
@@ -0,0 +1,348 @@
+#define _LOCK_EXTENSION
```

Comment:
Can you please add a short comment on each file (at least the ones that are not found in other ports) what those files are about? That helps a lot for reviews ;-)

Reply:
If I know :) The original port was written by @benavento a long time ago.